Dockerfile

```diff
@@ -3,7 +3,7 @@
 
 FROM alpine:3.18
 
-LABEL maintainer="OpenTF Team <opentf@opentf.org>"
+LABEL maintainer="OpenTofu Team <opentf@opentf.org>"
```

Comment:
Please rollback as it is being handled by https://github.com/opentofu/opentofu/pull/456